examples/Remarketing/AddFlightsFeed.php

```diff
@@ -0,0 +1,423 @@
+            'type' =>  FeedAttributeType::STRING,
+            'name' => new StringValue(['value' => 'Flight Price'])
+        ]);
+        // Creates a flight sale price attribute.
+        $flightSalesPriceAttribute = new FeedAttribute([
+            'type' =>  FeedAttributeType::STRING,
+            'name' => new StringValue(['value' => 'Flight Sale Price'])
+        ]);
+        // Creates a Final URLs attribute.
+        $finalUrlsAttribute = new FeedAttribute([
+            'type' =>  FeedAttributeType::URL_LIST,
+            'name' => new StringValue(['value' => 'Final URLs'])
+        ]);
+
+        // Creates the feed.
+        $feed = new Feed([
+            'name' => new StringValue(['value' => 'Flights Feed #' . uniqid()]),
+            'attributes' => [
+                $flightDescriptionAttribute,
+                $destinationIdAttribute,
+                $flightPriceAttribute,
+                $flightSalesPriceAttribute,
+                $finalUrlsAttribute
+            ]
+        ]);
+
+        // Creates the feed operation.
+        $operation = new FeedOperation();
+        $operation->setCreate($feed);
+
+        // Issues a mutate request to add the feed and print some information.
+        $feedServiceClient = $googleAdsClient->getFeedServiceClient();
+        $response = $feedServiceClient->mutateFeeds($customerId, [$operation]);
+        $feedResourceName = $response->getResults()[0]->getResourceName();
+        printf("Feed with resource name '%s' was created.%s", $feedResourceName, PHP_EOL);
+
+        return $feedResourceName;
+    }
+
+    /**
+     * Retrieves details about a feed. The initial query retrieves the feed attributes, or columns,
+     * of the feed. Each feed attribute will also include the feed attribute ID, which will be used
+     * in a subsequent step. The example then inserts a new key, value pair into a map for each
+     * feed attribute, which is the return value of the method. The keys are the placeholder types
+     * that the columns will be. The values are the feed attributes.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param string $feedResourceName the feed resource name to get its attributes
+     * @return array the map from placeholder fields to feed attributes
+     */
+    private static function getFeed(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $feedResourceName
+    ) {
+        $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+        // Constructs the query to get the feed attributes for the specified feed resource name.
+        $query = "SELECT feed.attributes FROM feed WHERE feed.resource_name = '$feedResourceName'";
+        // Issues a search request by specifying page size.
+        $response =
+            $googleAdsServiceClient->search($customerId, $query, ['pageSize' => self::PAGE_SIZE]);
```

Comment:
Why  not using a page size of 1 instead?

Reply:
Good question. This is done in the golden PR. I guess it's just copied over from other examples.
A bigger question is whether we want to micromanage this that much. 
Large page size should cause no harm in this case, no?

One small advantage I see is when people copy this code to their application , they don't need to worry much about too small page size.
In all examples, we use this same style over and over, so people are already used to it. They can just expect that they can configure PAGE_SIZE by their owns after copying the code.

Reply:
If to request more than expected is not considered as a bad practice then I believe it is good to keep it like that for the reasons you highlighted.